src/Servers/Connections.Abstractions/src/IConnectionListenerFactory.cs

```diff
@@ -10,8 +10,17 @@
 
 namespace Microsoft.AspNetCore.Connections
 {
+    /// <summary>
+    /// Defines an interface that provides the mechanisms for binding to various types of <see cref="EndPoint"/>s.
+    /// </summary>
     public interface IConnectionListenerFactory
     {
+        /// <summary>
+        /// Creates an <see cref="IConnectionListener"/> bound to the specified <see cref="EndPoint"/>.
+        /// </summary>
+        /// <param name="endpoint">The endpoint to bind to.</param>
```

Comment:
```suggestion
        /// <param name="endpoint">The <see cref="EndPoint" /> to bind to.</param>
```

Since we're talking about a specific type here...